library/src/morpheus-master/bootstrap-sass-3.3.7/assets/stylesheets/bootstrap/_jumbotron.scss

```diff
@@ -22,7 +22,7 @@
   }
 
   > hr {
-    border-top-color: darken($jumbotron-bg, 10%);
+    border-top-color: rgba(0,0,0,0.5);
```

Comment:
This seems a backward step, moving from a themed variable to something hardcoded. Is there a reason you've moved away from the variable?

Reply:
I can't use the variable on the bootstrap scss file right now